bundles/ApplicationLoggerBundle/src/Controller/LogController.php

```diff
@@ -54,14 +54,30 @@ public function onKernelControllerEvent(ControllerEvent $event): void
      */
     public function showAction(Request $request, Connection $db): JsonResponse
     {
+        $requestSource = $request->request;
+
+        //TODO: Remove the GET method support in Pimcore 12
+        if ($request->isMethod('GET')) {
+            trigger_deprecation(
+                'pimcore/pimcore',
+                '11.5.0',
+                sprintf('Calling route "%s" (%s) via "GET" method is deprecated and will not be supported anymore in 12.
+                Please use "POST" method instead.',
+                    'pimcore_admin_bundle_applicationlogger_log_show',
+                    __METHOD__
+                )
+            );
+            $requestSource = $request->query;
```

Comment:
should i use the allParams approach instead?
https://github.com/pimcore/pimcore/blob/4c36732719117f241336ce6515c1a6261d22a965/bundles/SimpleBackendSearchBundle/src/Controller/SearchController.php#L70